statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -941,6 +941,8 @@ def simulate_var(self, steps=None, offset=None, seed=None):
             the linear predictor of those components will be used as offset.
             This should have the same number of rows as steps, and the same
             number of columns as endogenous variables (neqs).
+        initial_state : None or ndarray (nlags, neqs)
```

Comment:
Same constraints here. Either (nlags, neqs) or (neqs,).  Type should be array_like, optional.